commands/init.ts

```diff
@@ -163,21 +160,10 @@ export async function handler(
     );
   }
 
-  const doesOtherConfigFileExist = configFileExists(!useHiddenConfig);
-  if (doesOtherConfigFileExist) {
-    const path = getConfigPath('', !useHiddenConfig);
-    logger.error(
-      i18n(`${i18nKey}.errors.bothConfigFilesNotAllowed`, { path: path! })
-    );
-    process.exit(EXIT_CODES.ERROR);
-  }
-
```

Comment:
Figured we could simplify this logic a bit. If _any_ config file exists (which we check for above), exit regardless of whether its local or global. I updated the error message to make it make sense for both cases